CMakeLists.txt

```
@@ -286,7 +282,7 @@ if (PATCHESTRY_INSTALL)
   # Executables are handled in the respective tools/CMakeLists.txt files.
   # TODO: Should we be moving over html docs and other files?
   install(DIRECTORY ${PE_VENDOR_INSTALL_DIR}/
-    DESTINATION ${CMAKE_INSTALL_LIBDIR}/../
+    DESTINATION ${CMAKE_INSTALL_PREFIX}
```

This is better as it's not hard-coded to the cache variable
```suggestion
    DESTINATION .
```
This matters if you do
```
cmake --install build --prefix blah
```
because the `--prefix` can set things dynamically